### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by fortanza**

### CODE REVIEW

**Review by John-athan14 — DISMISSED** [review]

**Review by John-athan14 — DISMISSED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -11,7 +12,8 @@ function App() {
       <Header />
       <Switch>
         <Route exact path="/" component={Home} />
-        <Route exact path="/Cards/" component={CardDescription} />
+        <Route path="/cards/:needle" component={SearchCards} />
+        <Route path="/Card/" component={CardDescription} />
```
Comment (Hacene010) [inline src/App.jsx]: why the uppercase for 'Card' ? and the "/" in the end ?

File: src/Components/SectionCarousel/SectionCarousel.jsx [hunk 2]
```diff
@@ -2,7 +2,8 @@ import './SectionCarousel.css';
 
 const SectionCarousel = () => {
   return (
-    <section className="SlideCaroussel">
+      <section className="SlideCaroussel">
```
Comment (Hacene010) [inline src/Components/SectionCarousel/SectionCarousel.jsx]: Use camelCase for the calssName

File: src/Components/SectionSearchbar/SearchBar/SearchBar.jsx [hunk 3]
```diff
@@ -1,17 +1,38 @@
...
-        />
+        <form onSubmit={HandlerDiplayCardsContainer}>
+          <input
+            type="search"
+            className="searchTerm"
+            placeholder="Search..."
+            onChange={HandlerChangeValue}
+          />
+          <img
+            className="input-logo"
```
Comment (Hacene010) [inline src/Components/SectionSearchbar/SearchBar/SearchBar.jsx]: camelCase plz

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Correct some bugs

**Review by John-athan14 — APPROVED** [review]

File: src/Components/SectionSearchbar/SearchBar/SearchBar.jsx (hunk 3 above)
Comment (fortanza) [reply src/Components/SectionSearchbar/SearchBar/SearchBar.jsx]: Problem resolved!

File: src/Components/SectionCarousel/SectionCarousel.jsx (hunk 2 above)
Comment (fortanza) [reply src/Components/SectionCarousel/SectionCarousel.jsx]: Problem resolved!

File: src/App.jsx (hunk 1 above)
Comment (fortanza) [reply src/App.jsx]: Problem resolved!

**Review by John-athan14 — DISMISSED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Hacene010) [reply src/App.jsx]: I was asking for camelCase not PascalCase : ).

File: src/Components/SectionSearchbar/SearchBar/SearchBar.css [hunk 4]
```diff
@@ -20,7 +20,7 @@ h1 {
   box-shadow: 1px 1px 2px #d4ecdd;
 }
 
-.input-logo {
+.InputLogo {
```
Comment (Hacene010) [inline src/Components/SectionSearchbar/SearchBar/SearchBar.css:23]: Same here. (We use PascalCase only for the components names)

File: src/App.jsx (hunk 1 above)
Comment (Hacene010) [reply src/App.jsx]: And don't foget to remove the slash after the "/card"

File: src/Components/SectionSearchbar/SearchBar/SearchBar.jsx [hunk 5]
```diff
@@ -1,17 +1,39 @@
...
-        />
+        <form onSubmit={HandlerDiplayCardsContainer}>
+          <input
+            type="search"
+            className="searchTerm"
+            placeholder="Search..."
+            onChange={HandlerChangeValue}
+          />
+          <img
+            className="InputLogo"
```
Comment (Hacene010) [inline src/Components/SectionSearchbar/SearchBar/SearchBar.jsx:32]: camelCase plz

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Saw some evils

File: src/App.jsx (hunk 1 above)
Comment (fortanza) [reply src/App.jsx]: / remove

**Review by John-athan14 — APPROVED** [review]

File: src/App.jsx (hunk 1 above)
Comment (fortanza) [reply src/App.jsx]: camelCase in the link  is changed. Can you validate your pull request please.

**Review by Hacene010 — APPROVED** [review]
Seems ok for me

**Pull request merged**